### ISSUE TITLE
GM_addStyle undefined

### ISSUE THREAD

**Issue by SolidTux** [EvHaus/youtube-hide-watched#16]
When I try to use the script, I get the following error:

```
ReferenceError: GM_addStyle is not defined
Stack trace:
userScript/<@user-script:http%3A//www.globexdesigns.com//YouTube%3A%20Hide%20Watched%20Videos:387:5
userScript@user-script:http%3A//www.globexdesigns.com//YouTube%3A%20Hide%20Watched%20Videos:380:2
scopeWrapper@user-script:http%3A//www.globexdesigns.com//YouTube%3A%20Hide%20Watched%20Videos:648:9
@user-script:http%3A//www.globexdesigns.com//YouTube%3A%20Hide%20Watched%20Videos:361:17
```

It looks like this function doesn't exist anymore ([source](https://wiki.greasespot.net/GM_addStyle)).

Browser: Firefox 57.0.2 (64bit, Linux)
Greasemonkey: 4.0

**Comment by EvHaus** [comment]
I can't reproduce this issue. Can you try upgrading to Greasemonkey 4.1?

**Comment by Ysix** [comment]
I have the same error : 
GreaseMonkey 4.1
Firefox 57.0.1 (Mac)
Script Version : 2.5

**Comment by EvHaus** [comment]
This is fixed in v2.6 of the script. Get it now here: https://greasyfork.org/en/scripts/13040-youtube-hide-watched-videos
